frontend/components/Footer/index.tsx

```diff
@@ -1,17 +1,60 @@
+import {
+  Center,
+  Flex,
+  GridItem,
+  Heading,
+  HStack,
+  SimpleGrid,
+  Text,
+  useBreakpointValue,
+} from "@chakra-ui/react";
 import React from "react";
 
 import styles from "./Footer.module.css";
 
+export const footerLinks = [
+  "Terms",
+  "Privacy",
+  "Help",
+  "API",
+  "Status",
+  "Blog",
+];
+
 export const Footer = (): JSX.Element => {
+  const verticalOffset = useBreakpointValue({ sm: "", md: "6" });
+  const spacing = useBreakpointValue({ xs: 6, sm: 14 });
+
   return (
-    <footer className={styles.footer}>
-      <a
-        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        Hello{" "}
-      </a>
-    </footer>
+    <Flex className={styles.footer}>
```

Comment:
Suggestion: Since this component is used outside of the main component, we need a semantic sectional HTML element wrapping it, rather than a div. Can we wrap this in a footer?
```suggestion
    <Flex className={styles.footer} as="footer">
```
Reference: https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html

Reply:
Ah okay, just read up on the ARIA practices and accessible considerations! Does your fix automatically do the `role="footer"` tag onto the footer as well?

Reply:
It doesn't automatically add a role="footer", rather it's a replacement to it!
It's recommended to use semantic html elements over adding roles to ambigious elements. 
I believe just having it as a footer html element should be sufficient considering we're only using it at the end of the page, (if we're using footers in a different context then we'd have to specific what role it is!)

If I remember correctly, the default role of a footer is contentinfo, so the role would be that I think